hubblestack/utils/signing.py

```diff
@@ -352,44 +440,59 @@ def sign_target(fname, ofname, private_key='private.key', **kwargs): # pylint: d
         args['algorithm'] = utils.Prehashed(chosen_hash)
     sig = first_key.sign(**args)
     with open(ofname, 'w') as fh:
-        log.debug('writing signature of %s to %s', os.path.abspath(fname), os.path.abspath(ofname))
+        log.debug('writing signature of {} to {}'.format(os.path.abspath(fname), os.path.abspath(ofname)))
         fh.write(PEM.encode(sig, 'Detached Signature of {}'.format(fname)))
         fh.write('\n')
 
 def verify_signature(fname, sfname, public_crt='public.crt', ca_crt='ca-root.crt', **kwargs): # pylint: disable=unused-argument
+    ### make 
     """
         Given the fname, sfname public_crt and ca_crt:
 
         return STATUS.FAIL if the signature doesn't match
         return STATUS.UNKNOWN if the certificate signature can't be verified with the ca cert
         return STATUS.VERIFIED if both the signature and the CA sig match
     """
-    log.debug("verify_signature(fname=%s, sfname=%s, public_crt=%s, ca_crt=%s", fname, sfname, public_crt, ca_crt)
+    log_level = log.debug
+    short_fname = fname.split('/')[-1]
     try:
         with open(sfname, 'r') as fh:
             sig,_,_ = PEM.decode(fh.read()) # also returns header and decrypted-status
     except IOError:
-        log.info('verify_signature() failed to find sfname=%s for fname=%s', sfname, fname)
+        status = STATUS.UNKNOWN
+        verif_key = ':'.join([fname, sfname])
+        if check_verif_timestamp(verif_key) == True:
+            log_level = log.error
+        msg = '{} | file "{}" | status: {} '
+        log_level(msg.format(short_fname, fname, status))
         return STATUS.UNKNOWN
     x509 = X509AwareCertBucket(public_crt, ca_crt)
     hasher, chosen_hash = hash_target(fname, obj_mode=True)
     digest = hasher.finalize()
+
     args = { 'signature': sig, 'data': digest }
     for crt,txt,status in x509.public_crt:
-        log.debug('trying to check %s with %s', sfname, txt)
+        log_level = log.debug
+        sha256sum = hash_target(fname)
+        msg = '{} | file "{}" | status: {} | sha256sum: "{}" | public cert fingerprint and requester: "{}"'
         pubkey = crt.get_pubkey().to_cryptography_key()
         if isinstance(pubkey, rsa.RSAPublicKey):
             args['padding'] = padding.PSS( mgf=padding.MGF1(hashes.SHA256()),
                 salt_length=padding.PSS.MAX_LENGTH)
             args['algorithm'] = utils.Prehashed(chosen_hash)
         try:
             pubkey.verify(**args)
+            log_level(msg.format(short_fname, fname, status, sha256sum, txt ))
             return status
         except InvalidSignature:
+            status = STATUS.FAIL
+            if check_verif_timestamp(fname) == True:
```

Comment:
Redundant comparison to True, check_verif_timestamp() returns bool value already.